src/index.js

```diff
@@ -380,7 +401,14 @@ function graphqlHTTP(options: Options): Middleware {
           );
         }
 
-        formatErrorFn = optionsData.formatError;
+        if (optionsData.formatError) {
+          console.error(
+            '`formatError` is deprecated and replaced by `customFormatErrorFn`. It will be removed in version 1.0.0.',
+          );
+        }
+
+        formatErrorFn =
+          optionsData.customFormatErrorFn || optionsData.formatError;
```

Comment:
@IvanGoncharov interestingly I couldn't change this to:

```
optionsData.customFormatErrorFn || optionsData.formatError || formatError
// ...
.map(errorFormatFn) 
```

Without breaking the tests. It appears the 

```
.map(errorFormatFn || formatError) 
```

may be called before the options resolve. maybe around schema validation. 

Might be worth resolving outside this change.